clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -228,6 +230,10 @@ func (w *waitForControlledPodsRunningMeasurement) gather(syncTimeout time.Durati
 		return fmt.Errorf("timed out while waiting for controlled pods")
 	}
 
+	k8sClient := w.clusterFramework.GetClientSets().GetClient()
+	forceDeletePodOptions := forceDeleteOptions()
+	listOptions := w.listOptions()
```

Comment:
This will be fairly rare case - let's not create it always - see my comment below.

Reply:
done